integration-tests/spring-data-jpa/pom.xml

```diff
@@ -87,10 +87,6 @@
                             <goal>test</goal>
                         </goals>
                         <configuration>
-                            <systemPropertyVariables>
-                                <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
-                            </systemPropertyVariables>
-                            <argLine>${jacoco.agent.argLine}</argLine>
```

Comment:
@geoand is this one necessary?

Reply:
FWIW, this is/was already in `build-parent`.

Reply:
I must have added it by copying some other configuration. But since it's in the build parent, there is no need to have it